src/documentation/runners/apex.md

```diff
@@ -5,5 +5,5 @@ permalink: /documentation/runners/apex/
 ---
 # Using the Apache Apex Runner
 
-This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)). The runner is on a feature branch.
+This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
```

Comment:
FYI only -- see #76 and #77 for good examples on getting started on this. (Not for this PR.)